intl/msg_hash_us.h

```diff
@@ -13668,6 +13668,50 @@ MSG_HASH(
    MSG_CHEEVOS_HARDCORE_MODE_ENABLE,
    "Achievements Hardcore Mode Enabled, save state & rewind were disabled."
    )
+MSG_HASH(
+   MSG_CHEEVOS_HARDCORE_MODE_PAUSED,
+   "Hardcore paused. You cannot earn hardcore achievements for %s using %s"
```

Comment:
The enum for this should be more specific. Maybe `MSG_CHEEVOS_HARDCORE_PAUSED_INVALID_CORE`?

There are several other reasons hardcore could be disabled in `rcheevos_validate_config_settings`.